.github/workflows/code_quality.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
           for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
             echo "$file was changed"
             if [[ $file == *.py ]]; then
-               black $file --check --diff
+               black $file --check --diff --line-length 120
```

Comment:
consider configuring this in pyproject.toml, https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#configuration-via-a-file

c.f.

https://github.com/red-hat-data-services/ods-ci/blob/7df1a217ca7668a3a4983f47f27995439864fc27/pyproject.toml#L48

https://github.com/red-hat-data-services/ods-ci/blob/7df1a217ca7668a3a4983f47f27995439864fc27/pyproject.toml#L52